.travis.yml

```diff
@@ -9,6 +9,7 @@ matrix:
       dist: trusty
     - php: 7.4
       dist: trusty
+    - php: nightly
```

Comment:
I saw that PHP 8 has been released now.
Should it be added to the list?
Do we keep nightly? Or is it likely to break things occasionally and block merging PRs?

Reply:
I committed a change to `php: 8.0`. Lets see how it goes.

Reply:
Looks good :+1: